packages/iris-grid/src/mousehandlers/IrisGridColumnSelectMouseHandler.ts

```diff
@@ -4,7 +4,7 @@ import {
   GridPoint,
   EventHandlerResult,
 } from '@deephaven/grid';
-import type { Column } from '@deephaven/jsapi-shim';
+import { Column } from '@deephaven/jsapi-types';
```

Comment:
Didn't need to remove the keyword `type` here.
```suggestion
import type { Column } from '@deephaven/jsapi-types';
```
Doing an `import type` over just a regular import could improve performance; we should pretty much always use `import type` when importing from `jsapi-types`